requirements.txt

```diff
@@ -1,12 +1,12 @@
-argcomplete==1.11.1
-colorama==0.4.3
-flake8==3.7.9
-jmespath==0.9.5
-mock==4.0.1
-pylint==2.5.3
-Pygments==2.5.2
-PyYAML==5.3.1
-six==1.14.0
-tabulate==0.8.6
+argcomplete==1.12.2
+colorama==0.4.4
+flake8==3.8.3
+jmespath==0.10.0
+mock==4.0.2
+pylint==2.3.0
+Pygments==2.7.4
+PyYAML==5.4.1
```

Comment:
This `requirements.txt` is actually never used. Instead, we always honor `setup.py`.

Now sure if this will trigger issues like https://github.com/Azure/azure-cli/issues/16650, so bump them anyway.